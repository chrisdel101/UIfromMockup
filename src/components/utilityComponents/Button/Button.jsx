import React from 'react'
import { Button } from 'react-bootstrap'

function myButton(props) {
  return <Button variant={props.variant}>{props.text}</Button>
}
export default myButton
