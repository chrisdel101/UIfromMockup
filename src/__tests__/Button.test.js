import React from 'react'
import Button from '../components/utilityComponents/Button/Button.jsx'
import renderer from 'react-test-renderer'

test('Link changes the class when hovered', () => {
  const component = renderer.create(<Button variant="primary">Facebook</Button>)
  console.log(component)
  // let tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
  // // manually trigger the callback
  // tree.props.onMouseEnter();
  // // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
  // // manually trigger the callback
  // tree.props.onMouseLeave();
  // // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
})
