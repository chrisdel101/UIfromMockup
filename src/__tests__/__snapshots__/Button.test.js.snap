// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create and match snapshot 1`] = `
<button
  className="btn btn-primary"
  disabled={false}
  type="button"
/>
`;
